src/logic/operate.js

```diff
@@ -0,0 +1,23 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const first = new Big(numberOne);
```

Comment:
numberOne, and numberTwo should be wrapped with Big class. Currently, we only have numberOne wrapped in Big class